apps/web/src/components/FeatureBlock/FeatureBlock.tsx

```diff
@@ -42,14 +56,15 @@ const FeatureBlock = ({ content }: FeatureBlockProps) => {
           {content.title}
         </Heading>
       </div>
-      <div
+      {/* <div
         data-h2-height="base(x10) desktop(x12)"
         style={{
           backgroundImage: `url('${content.img.path}')`,
           backgroundPosition: content.img.position || "center",
           backgroundSize: "cover",
         }}
-      />
+      /> */}
+      <ReactImage UrlArray={[content.img.path, content.img.fallback]} />
```

Comment:
For this instance of background image, I just tried replacing it with an image component, but in general not sure what is the best course of action for background images

Reply:
Like this is a background image and I haven't changed it yet

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/c75095b3-10d4-4c29-b4de-6e20b2a23eff)

![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/40485260/16de81ab-e73f-4fba-b3a7-3b216c589385)